articles/multifactor-authentication/guardian/admin-guide.md

```diff
@@ -0,0 +1,123 @@
+# Guardian for Administrators 
+
+Guardian is Auth0's multifactor authentication (MFA) application that provides a simple, safe way for you to implement MFA. The Guardian app is used for two-factor authentication when logging into an application, which helps create a more secure login.  With two-factor authentication your users will always need their mobile device in order to login.
+
+This page will help to explain how to enable and use Guardian for MFA for signing in your users.
+
+For information for your users on what Guardian is, how to download the app and common questions they may have, see [here](/guardian/user-guide).
+ 
+## Enabling Guardian 
+
+You can enable Guardian by going to the [Multifactor Auth](${uiURL}/#/guardian) section of the dashboard. By toggling the **Push Notification** slider, you can enable using Guardian for sign in and sign up for your application. 
+
+![](/media/articles/mfa/guardian-dashboard.png)
+
+New users signing up will be prompted to download the Guardian app from either the App Store or Google Play. Once they indicate that they downloaded the app a code will appear. They will have five minutes to scan the code with the app before it expires. After the code has been successfully scanned, users will see a confirmation screen which includes a recovery code. They need to have this recovery code to login without their mobile device. If they lose both the recovery code and their mobile device, you will need to [reset their MFA](/admin-guide#reset-mfa-for-a-user).
+
+Users that were previously registered before you enable MFA, will complete the same process as new users on their next login.
+
+By enabling **Push Notifications** for MFA, you will also be enabling the option to use Google Authenticator instead of Guardian. To use this option, your users must have the Google Authenticator app downloaded to their mobile device, the app is available for both Android and iOS. When a new user signs up, or a user is logging in for the first time since enabling MFA there will be a code to be scanned in the Google Authenticator app. After scanning the code, they will get a six digit code to enter. Once they enter this code, there will be a confirmation screen which has a recovery code. They need enter this recovery code to login without their mobile device. If they do not have the recovery code or their mobile device, you will need to [reset MFA](/admin-guide#reset-mfa-for-a-user) for their account.
```

Comment:
This paragraph is too complex and should be somwhere else, not on Guardian intro. Like a note saying that Guardian behind the scenes uses the same mechanism that Google Authenticator (or any TOTP Authenticator) uses, so for phones where there is no Guardian (e.g. Windows Phones), you can scan the QR code using Microsoft Authenticator. Actually, I would make a special section explaining the support on different types of devices and versions of OS.